app/Configuration/SetConfig.php

```diff
@@ -142,4 +116,30 @@ private function getWithOutput(string $key, $default): bool
 
         return $this->get($key, $default);
     }
+
+    public function __invoke($defaultConfiguration)
```

Comment:
Why is this below the private methods?

Reply:
(If this is a TLint rule, I think we need to fix it; I think of `__invoke` as being more like a constructor than the other magic methods, but maybe I'm crazy)

Reply:
@mattstauffer yeah, TLint!